### PR TITLE
Add log-level customization for artifact logs in E2E runs

### PR DESCRIPTION
**Pull request by Ankitasw**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
This PR adds log-level for artifacts logs of the E2E test runs which would be useful for development/debugging.

**Checklist**:
- [ ] squashed commits
- [ ] includes documentation
- [ ] adds unit tests
- [ ] adds or updates e2e tests

**Release note**:
```release-note
Controller log level can be set using the CAPA_LOGLEVEL variable. Defaults to 0.
```

### CODE REVIEW

File: config/manager/manager.yaml [hunk 1]
```diff
@@ -21,6 +21,7 @@ spec:
         - "--metrics-bind-addr=127.0.0.1:8080"
         - "--leader-elect"
         - "--feature-gates=EKS=${CAPA_EKS:=true},EKSEnableIAM=${CAPA_EKS_IAM:=false},EKSAllowAddRoles=${CAPA_EKS_ADD_ROLES:=false},EKSFargate=${EXP_EKS_FARGATE:=false},MachinePool=${EXP_MACHINE_POOL:=false},EventBridgeInstanceState=${EVENT_BRIDGE_INSTANCE_STATE:=false},AutoControllerIdentityCreator=${AUTO_CONTROLLER_IDENTITY_CREATOR:=true}"
+        - "--v=${EKS_CP_LOGLEVEL:=4}"
```
Comment (richardcase) [inline config/manager/manager.yaml:24]: Now we have merged the controllers together, it's not specific to EKS. 

perhaps we should change its name to `CAPA_LOGLEVEL`, wdyt?

**Review by richardcase — CHANGES_REQUESTED** [review]

File: config/manager/manager.yaml (hunk 1 above)
Comment (Ankitasw) [reply config/manager/manager.yaml:24]: sounds fine, will make the change

File: config/manager/manager.yaml [hunk 2]
```diff
@@ -21,6 +21,7 @@ spec:
         - "--metrics-bind-addr=127.0.0.1:8080"
         - "--leader-elect"
         - "--feature-gates=EKS=${CAPA_EKS:=true},EKSEnableIAM=${CAPA_EKS_IAM:=false},EKSAllowAddRoles=${CAPA_EKS_ADD_ROLES:=false},EKSFargate=${EXP_EKS_FARGATE:=false},MachinePool=${EXP_MACHINE_POOL:=false},EventBridgeInstanceState=${EVENT_BRIDGE_INSTANCE_STATE:=false},AutoControllerIdentityCreator=${AUTO_CONTROLLER_IDENTITY_CREATOR:=true}"
+        - "--v=${CAPA_LOGLEVEL:=4}"
```
Comment (randomvariable) [inline config/manager/manager.yaml:24]:
```suggestion
        - "--v=${CAPA_LOGLEVEL:=0}"
```

Let's leave the default as 0, and devs can bump it to 4 when they want to.

**Pull request merged**